samples/http-image-cloudevents/client.py

```diff
@@ -0,0 +1,84 @@
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import base64
+import io
+import json
+import sys
+import typing
+
+import requests
+from PIL import Image
+
+from cloudevents.sdk import converters
+from cloudevents.sdk.http_events import CloudEvent
+
+
+def create_byte_image(size: typing.Tuple[int, int]) -> str:
```

Comment:
Comment as to what these methods are doing?

Reply:
removed those methods